vm/neovm/func_stack_test.go

```diff
@@ -366,3 +367,16 @@ func TestOpTuck(t *testing.T) {
 		t.Fatal("NeoVM OpTuck test failed.")
 	}
 }
+
+func TestStruct(t *testing.T) {
+	s := types.NewStruct(nil)
+	k := types.NewStruct([]types.StackItems{types.NewInteger(big.NewInt(1))})
+	for i := 0; i < 1024; i++ {
```

Comment:
from your implementation. i think `1` is enough to lead the following assert to false.